js/config/styles.js

```diff
@@ -7,6 +8,25 @@ const globalStyles = {
     letterSpacing: 2
   },
 
+  titleFontLoginForgotPassword: {
+    color: '#5D5D5D',
+    alignSelf: 'center',
+    fontSize: 40,
+    fontFamily: 'Quicksand'
+  },
+  logoImageLoginForgotPassword: {
+    height: height / 5,
+    width: width / 4,
+    alignSelf: 'center',
+    resizeMode: 'center',
+    marginTop: 50,
+    marginBottom: -15
```

Comment:
we never assign negative margin and any huge margins like 50. Please change this and use flexbox to your advantage.